docs/changelog.rst

```diff
@@ -138,9 +143,11 @@ according to `SemVer <http://semver.org>`__. The following changes are
 noteworthy:
 
 * The :cpp:class:`nb::enum_\<T\>() <enum_>` binding declaration is now a
-  wrapper that creates either a ``enum.Enum`` or ``enum.IntEnum``-derived type.
+  wrapper that creates either a ``enum.Enum``, ``enum.IntEnum`` or ``enum.Flag``-derived type.
```

Comment:
I think it would make sense to support a matrix of 4 base classes: `nb::is_arithmetic` switches between `enum.*` and `enum.Int*` while `nb::is_flag` switches between `Enum` and `Flag`.